src/core/packages/saved-objects/common/src/saved_objects_imports.ts

```diff
@@ -61,6 +61,37 @@ export interface SavedObjectsImportUnsupportedTypeError {
   type: 'unsupported_type';
 }
 
+// Note: the following two interfaces are not needed until we support importing
```

Comment:
I'm worried we might forget about commented out code and delete it accidentally along the way. Can we add a link to a GH issue as a reference? It'll be easier to follow than git blame